tests/safety/common.py

```diff
@@ -175,6 +174,53 @@ def test_accel_actuation_limits(self, stock_longitudinal=False):
           self.assertEqual(should_tx, self._tx(self._accel_msg(accel)))
 
 
+class LongitudinalGasBrakeSafetyTest(PandaSafetyTestBase, abc.ABC):
```

Comment:
New PR for making this test. The only other brand I saw that was using gas/brake was GM, do you know of any others?

https://github.com/commaai/panda/pull/1537

Reply:
Think honda might? Also Ford. Though it's a bit different, they implement a gas check and accel check (for braking only). Might want classes to individually test each, or a class for these two cases as it seems common.

Reply:
Though since we don't have a nice way to know the max ranges of signals, you can keep the gas/brake tests locally to Subaru for now, and we can clean up in a future PR.